docs/vm-ee-guide.md

```diff
@@ -0,0 +1,324 @@
+# Onboard
```

Comment:
Why we have Onboard section at the very top.. Also, come up with better name for the document.. vm-ee-guide doesnt infer anything